@xen-orchestra/web-core/lib/components/fullscreenable/FullscreenableHeader.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <header class="header">
+    <UiIcon :icon="faAngleLeft" color="info" @click="back" />
```

Comment:
According to the design system, it should be a `ButtonIcon` component:
```suggestion
    <ButtonIcon :icon="faAngleLeft" color="info" @click="back" />
```